saleor/product/models.py

```diff
@@ -399,6 +400,18 @@ def save(
             self.token = str(uuid4()).replace("-", "")
         super().save(force_insert, force_update, using, update_fields)
 
+    def increment_download_count(self):
```

Comment:
I'd move it to a utility function to keep our models thin. It's okay for a model method to perform simple logic on model's fields but if it requires saving additional models, then we usually keep that in a function outside.

Reply:
It's planned in the checklist! 👍  Thanks!